README.md

```diff
@@ -1,13 +1,35 @@
-# create-codes monorepo
+# ML-frontend
 
-This is monorepo for creating new front-end codes with a standard structure.
+ML-frontend is a Monorepo created to provide a starter kit for Front-end projects at Monstar Lab, Inc.<br/>
```

Comment:
Some minor suggestions:
1. Since it's a name, should the "frontend" in the repository name be capitalized? `ML-Frontend`
2. Besides the title, "front-end" should be small caps.
```
... a starter kit for front-end projects...
```
```
... for modern front-end development.
```

3. I think "monorepo" should also be small caps.